src/charm.py

```diff
@@ -83,7 +66,7 @@ def __init__(self, framework: ops.Framework):
             for event in [
                 self.on["gocert"].pebble_ready,
                 self.on["gocert"].pebble_custom_notice,
-                self.tls.on.certificate_creation_request,
+                self.on.certificates_relation_changed,
```

Comment:
I'd like to keep the formats here the same.
```suggestion
                self.on["certificates"].relation_changed,
```